packages/superset-ui-query/src/types/Datasource.ts

```diff
@@ -10,8 +10,14 @@ export enum DatasourceType {
 export interface Datasource {
   id: number;
   name: string;
-  description?: string;
   type: DatasourceType;
   columns: Column[];
   metrics: QueryObjectMetric[];
+  description?: string;
+  columnFormats?: {
+    [key: string]: string;
```

Comment:
could we be any more specific about what `key` is here + `verboseMap` below? like `name` or `id` or something?

Reply:
I always thought it's good to be generic in indexable types so that new TypeScript users don't confuse it with an actual property name.

Reply:
I'll add some inline comments instead.